pipelines/upgrade_pipeline.yml

```diff
@@ -4,7 +4,9 @@
 - import_playbook: upgrade/03-install_server.yaml
 - import_playbook: upgrade/04-install_proxy.yaml
 - import_playbook: upgrade/05-server_to_intermediate.yaml
+  when: forklift_upgrade_version_start != forklift_upgrade_version_intermediate
```

Comment:
One issue is that in CI it doesn't actually run this playbook but rather all individual ones

Reply:
oh. right. because we are clever.

Reply:
how about now?

Reply:
(it wouldn't have worked anyways, as the vars are only loaded inside the playbook)